src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -69,6 +68,56 @@ export class CommentEditorComponent implements OnInit {
     this.commentField.setValidators([Validators.maxLength(this.maxLength)]);
   }
 
+  onKeyPress(event) {
```

Comment:
`onKeyPress` as the method name describes should handle switch-case for key presses, then call private methods.